src/gridInterceptor/interceptorFunctions.js

```diff
@@ -1,147 +1,153 @@
-            if (objKeyItem.localeCompare('type')) {
-              objectListItem.innerHTML += objKeyItem + ' = ' + objArrayItem[objKeyItem] + ' ';
-            } else {
-              objectListItem.innerHTML += objArrayItem[objKeyItem] + ' ';
-            }
-          }
-        });
-      });
-      object2.objectArray.forEach(function(objArrayItem,i){
-        var objectListItem = document.createElement('li');
-        objectListItem.id = 'object' + (object1.objectArray.length + i);
-        objectList.appendChild(objectListItem);
-        var objKeys = Object.keys(objArrayItem.getAttributes());
-        objKeys.forEach(function(objKeyItem){
-          if (objKeyItem.localeCompare('coordLoc')) {
-            if (objKeyItem.localeCompare('type')) {
-              objectListItem.innerHTML += objKeyItem + ' = ' + objArrayItem[objKeyItem] + ' ';
-            } else {
-              objectListItem.innerHTML += objArrayItem[objKeyItem] + ' ';
-            }
-          }
-        });
-      });
-      elementDetail.appendChild(objectList);
+        if (this.totalCount < 100) {
+            object1.objectArray.forEach(function(objArrayItem, i) {
+                let objectListItem = document.createElement(`li`);
+                objectListItem.id = `object` + i;
+                objectList.appendChild(objectListItem);
+                let objKeys = Object.keys(objArrayItem.getAttributes());
+                objKeys.forEach(function(objKeyItem) {
+                    if (objKeyItem.localeCompare(`coordLoc`)) {
+                        if (objKeyItem.localeCompare(`type`)) {
+                            objectListItem.innerHTML += objKeyItem + ` = ` + objArrayItem[objKeyItem] + ` `;
+                        } else {
+                            objectListItem.innerHTML += objArrayItem[objKeyItem] + ` `;
+                        }
+                    }
+                });
+            });
+            object2.objectArray.forEach(function(objArrayItem, i) {
+                let objectListItem = document.createElement(`li`);
+                objectListItem.id = `object` + (object1.objectArray.length + i);
+                objectList.appendChild(objectListItem);
+                let objKeys = Object.keys(objArrayItem.getAttributes());
+                objKeys.forEach(function(objKeyItem) {
+                    if (objKeyItem.localeCompare(`coordLoc`)) {
+                        if (objKeyItem.localeCompare(`type`)) {
+                            objectListItem.innerHTML += objKeyItem + ` = ` + objArrayItem[objKeyItem] + ` `;
+                        } else {
+                            objectListItem.innerHTML += objArrayItem[objKeyItem] + ` `;
+                        }
+                    }
+                });
+            });
+            elementDetail.appendChild(objectList);
+        }
     }
-  }
 }
 
 
-var gridInterceptor = new gridInterceptor();
+const gridInterceptor = new gridInterceptor();
```

Comment:
Using Pascal case this shouldn't be an issue. Can you change it to Pascal case?

Reply:
This has been changes in the last commit